Xamarin.Essentials/Sms/Sms.shared.cs

```diff
@@ -18,18 +20,14 @@ public static Task ComposeAsync(SmsMessage message)
 
     public class SmsMessage
     {
-        public SmsMessage()
-        {
-        }
-
-        public SmsMessage(string body, string recipient)
+        public SmsMessage(string body, string[] recipients)
         {
             Body = body;
-            Recipient = recipient;
+            Recipients = recipients.Where(x => !string.IsNullOrWhiteSpace(x)).ToArray();
         }
 
         public string Body { get; set; }
 
-        public string Recipient { get; set; }
+        public string[] Recipients { get; }
```

Comment:
Can we make this a `List<T>` and keep it settable?

`public List<string> Recipients  { get; set; } = new List<string>();`

I would like to be able to do:

```csharp
var msg = new SmsMessage {
  Recipients = SomeList
};
```
AND
```csharp
var msg = new SmsMessage {
  Recipients = {
    "1234",
    "5678"
  }
};
```

Reply:
Done